hpi/cloud/courses/v1test/course.proto

```diff
@@ -0,0 +1,109 @@
+syntax = "proto3";
+
+package hpi.cloud.courses.v1test;
+
+// A series of courses that cover the same topic and can take place on multiple
+// semesters.
+message CourseSeries {
+    // Required, output only. The unique ID of this course.
+  string id = 1;
+
+  // Required. The title.
+  string title = 2;
+
+  // Required. A short version of the title.
+  string short_title = 3;
+
+  // Required. An abbreviation of the title.
+  string abbreviation = 4;
+
+  // Required. The number of ECTS that the courses of this series give to the
+  // participants.
+  sint32 ects = 5;
+
+  // Required. The hours per week that courses of this series typically require.
+  sint32 hoursPerWeek = 6;
+
+  // Required. Whether it's mandatory to particiapte in a course of this series.
+  bool mandatory = 7;
+
+  // The language courses of this series are taught in.
+  string language = 8;
```

Comment:
What values are possible? Probably limit to [BCP 47](https://en.wikipedia.org/wiki/IETF_language_tag)